plugin/parser/ast/column_name_def.go

```diff
@@ -0,0 +1,9 @@
+package ast
+
+// ColumnNameDef is the struct for column name definition.
+type ColumnNameDef struct {
+	expression
+
+	Table      *TableDef
```

Comment:
Since we traverse from top to bottom, do we need to keep the reference to TableDef from ColumnNameDef? Or is this an inherited pattern from other places?

Reply:
Yeah, I think we’ll end up like this. But now, we don’t convert the FROM clause, so we can not do it here